handsontable/src/helpers/dom/element.js

```diff
@@ -969,6 +969,28 @@ export function setCaretPosition(element, pos, endPos) {
 
 let cachedScrollbarWidth;
 
+/**
+ * Returns the fractional scaling compensation for scrollbar width calculation.
+ *
+ * @param {Document} rootDocument The onwer of the document.
+ * @returns {number} The compensation for the scrollbar width, when the device pixel ratio is not an integer.
+ */
+// eslint-disable-next-line no-restricted-globals
+export function getScrollbarFractionalScalingCompensation(rootDocument = document) {
+  const rootWindow = rootDocument.defaultView;
+  const isWindows = rootWindow.navigator.userAgent.toLowerCase().includes('windows');
+
+  if (!isWindows) {
+    return 0;
+  }
```

Comment:
```suggestion

  if (!isWindowsOS()) {
    return 0;
  }
```
There is already a helper available. I highly encourage you to use it https://github.com/handsontable/handsontable/blob/feature/dev-issue-2427/handsontable/src/helpers/browser.js#L134.